WeakAuras/Types.lua

```diff
@@ -1320,6 +1320,32 @@ for id, str in pairs(Private.combatlog_spell_school_types) do
   Private.combatlog_spell_school_types_for_ui[id] = ("%.3d - %s"):format(id, str)
 end
 
+Private.item_quality_types = {
+  [0] = ITEM_QUALITY0_DESC,
+  [1] = ITEM_QUALITY1_DESC,
+  [2] = ITEM_QUALITY2_DESC,
+  [3] = ITEM_QUALITY3_DESC,
+  [4] = ITEM_QUALITY4_DESC,
+  [5] = ITEM_QUALITY5_DESC,
+  [6] = ITEM_QUALITY6_DESC,
+  [7] = ITEM_QUALITY7_DESC,
+  [8] = ITEM_QUALITY8_DESC,
+}
+
+Private.discovered_currencies = {}
+Private.discovered_currencies_sorted = {}
+for index = 1, C_CurrencyInfo.GetCurrencyListSize() do
+  local CurrencyInfo = C_CurrencyInfo.GetCurrencyListInfo(index)
+  local currencyLink = C_CurrencyInfo.GetCurrencyListLink(index)
+  if currencyLink then
+     local currencyID = C_CurrencyInfo.GetCurrencyIDFromLink(currencyLink)
+     Private.discovered_currencies[currencyID] = CurrencyInfo.name
+     tinsert(Private.discovered_currencies_sorted, CurrencyInfo.name)
+  end
+end
+Private.discovered_currencies["member"] = "|Tinterface\\common\\ui-searchbox-icon:0:0:0:-2|t"..L["Specific Currency"];
+Private.discovered_currencies_sorted["member"] = "|Tinterface\\common\\ui-searchbox-icon:0:0:0:-2|t"..L["Specific Currency"];
```

Comment:
`Private.discovered_currencies_sorted` is not used, and if it did then this line should be `tinsert(Private.discovered_currencies_sorted, "member")`